lib/tasks/evm_dba.rake

```diff
@@ -132,43 +64,14 @@ namespace :evm do
       exit 1
     end
 
+    # this is used by the appliance console to create a db
     # Example usage:
-    #   RAILS_ENV=production bin/rake evm:db:region -- --region 99
+    #   RAILS_ENV=production REGION=99 bin/rake evm:db:region
+    # Alt usage:
+    #   RAILS_ENV=production REGION=99 VERBOSE=false bin/rake db:reset db:seed
 
     desc 'Set the region of the current ManageIQ EVM Database (VMDB)'
-    task :region do
-      opts = EvmDba.with_options do
-        opt :region, "Region number", :type => :integer, :required => ENV["REGION"].blank?
-      end
-
-      Dir.chdir(Rails.root)
-      begin
-        #TODO: Raise an error if region is not valid
-        ENV["REGION"] = opts[:region].to_s if opts[:region]
-        region = ENV["REGION"]
-
-        region_file = Rails.root.join("REGION")
-        if File.exist?(region_file)
-          old_region = File.read(region_file)
-          File.delete(region_file)
-        end
-
-        puts "Resetting #{Rails.env} database to region #{region}..."
-        ENV['VERBOSE'] = 'false' # Do not flood the output with migration details
-        Rake::Task['evm:db:reset'].invoke
-
-        puts "Initializing region and database..."
-        # Create the region from our REGION file, initialize a new miq_database row for this region
-        AwesomeSpawn.run!("bin/rails runner", :params => ["MiqDatabase.seed; MiqRegion.seed"])
-      rescue => err
-        message = err.kind_of?(AwesomeSpawn::CommandResultError) ? err.result.error : err.message
-        STDERR.puts "Encountered issue setting up Database using region #{region}: #{message}\n"
-        File.write(region_file, old_region) if old_region
-        raise
-      end
-
-      exit # exit so that parameters to the first rake task are not run as rake tasks
-    end
+    task :region => %w[evm:db:reset db:seed]
```

Comment:
This is slightly different - The old code _only_ did `MiqDatabase.seed; MiqRegion.seed` not a full `db:seed`.  I recall that being intentional because we didn't want to incur the time penalty for a full db seed when it's going to run again later on.

Reply:
ok, I will roll this part of the code back.

Still think we should just do stuff the standard rails way rather than introducing nuance like having 2 layers of seeding.

Also, still unsure why we are seeding later when the database was just brought up to date. (old conversation about only seeding on database setup, database restore, code change)